### ISSUE TITLE
MA0097 - inherited classes

### ISSUE THREAD

**Issue by dgaspar** [meziantou/Meziantou.Analyzer#267]
**Context**
Rule identifier: MA0097
Target Framework: netstandard2.0
Language version: 8

**Describe the bug**
`abstract class Test` implements all the `IComparable` methods, but the analyzer complains that inherited classes aren't implementing them.
Would this be by design?

**To Reproduce**
````c#
    abstract class Test : IComparable // ok
    {
        public int CompareTo(object other) => 0;
        public override bool Equals(object? obj) => true;
        public override int GetHashCode() => 0;
        public static bool operator <(Test a, Test b) => true;
        public static bool operator <=(Test a, Test b) => true;
        public static bool operator >(Test a, Test b) => true;
        public static bool operator >=(Test a, Test b) => true;
        public static bool operator ==(Test a, Test b) => true;
        public static bool operator !=(Test a, Test b) => true;
    }

    class InheritedTest : Test // nok
    {
    }
````

**Additional context**
One example is this [`Enumeration` seedwork](https://github.com/dotnet-architecture/eShopOnContainers/blob/dev/src/Services/Ordering/Ordering.Domain/SeedWork/Enumeration.cs#L8). It wouldn't be ideal to have to copy-paste the implementation in every class inherited from `Enumeration`.

**Comment by meziantou** [comment]
The next version (1.0.676) should fix this issue
